python/grass/script/setup.py

```diff
@@ -24,17 +24,24 @@
     # query GRASS itself for its GISBASE
     # (with fixes for specific platforms)
     # needs to be edited by the user
-    grass8bin = 'grass'
-    if sys.platform.startswith('win'):
+    grass8bin = "grass"
+    if sys.platform.startswith("win"):
         # MS Windows
-        grass8bin = r'C:\OSGeo4W\bin\grass.bat'
+        grass8bin = r"C:\OSGeo4W\bin\grass.bat"
         # uncomment when using standalone WinGRASS installer
         # grass8bin = r'C:\Program Files (x86)\GRASS GIS 8.0.0\grass.bat'
         # this can be avoided if GRASS executable is added to PATH
-    elif sys.platform == 'darwin':
+    elif sys.platform == "darwin":
         # Mac OS X
-        # TODO: this have to be checked, maybe unix way is good enough
-        grass8bin = '/Applications/GRASS/GRASS-8.0.app/'
+        mac_gisbase = "/Applications/GRASS-8.0.app/Contents/Resources"
```

Comment:
The `mac_` part is not needed here since this is already under macOS, but I hope the whole variable is not needed.